check_header.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) FIRST and other WPILib contributors.
+# Open Source Software; you can modify and/or share it under the terms of
+# the WPILib BSD license file in the root directory of this project.
+#
+
+from pathlib import Path
+
+
+def check_file_content(file_path):
+    with open(file_path, "r") as file:
+        lines = file.readlines()
+
+        if file.name.endswith("robot.py"):
+            expected_lines = [
+                "#!/usr/bin/env python3\n",
+                "#\n",
+                "# Copyright (c) FIRST and other WPILib contributors.\n",
+                "# Open Source Software; you can modify and/or share it under the terms of\n",
+                "# the WPILib BSD license file in the root directory of this project.\n",
+                "#\n",
+                "\n",
+            ]
+        elif file.name.endswith("__init__.py"):
+            expected_lines = [
```

Comment:
I think you could get rid of the special case for this by just ignoring empty files.

Reply:
Good call.  And a super easy implementation too.  Thanks!